common/internal_model/src/test/scala/weco/catalogue/internal_model/work/AvailabilityTest.scala

```diff
@@ -28,13 +28,47 @@ class AvailabilityTest
       every(availabilities) should contain only Availability.Online
     }
 
-    it("adds Availability.InLibrary if there is a physical location") {
+    it("adds Availability.InLibrary if there is an item with a physical location") {
       val work = denormalisedWork().items(List(createIdentifiedPhysicalItem))
       val workAvailabilities = Availabilities.forWorkData(work.data)
 
       workAvailabilities should contain only Availability.InLibrary
     }
 
+    describe("if there is a holdings") {
```

Comment:
This is grammatically stressful

Reply:
evergreen description of the holdings work